vagrant/config/versions.yaml

```diff
@@ -93,6 +93,17 @@ installers:
       - 'debian12'
       - 'ubuntu2204'
 
+  - foreman: '3.16'
+    katello: '4.18'
+    candlepin: '4.6'
+    pulpcore: '3.73'
+    puppet: 8
+    boxes:
+      - 'almalinux9'
+      - 'centos9-stream'
+      - 'debian12'
+      - 'ubuntu2204'
+
   - foreman: 'nightly'
     katello: 'nightly'
     candlepin: '4.4'
```

Comment:
A bit outside the scope I know, but do you mind bumping this to 4.6 as well?

Reply:
updated